thrift.js

```diff
@@ -194,6 +197,8 @@ Thrift.prototype.compile = function compile(source) {
     assert.equal(syntax.type, 'Program', 'expected a program');
     this._compile(syntax.headers);
     this._compile(syntax.definitions);
+    this.compileEnum(messageExceptionTypesDef);
+    this.exception = this.compileStruct(messageExceptionDef);
```

Comment:
Would be nice to encapsulate the "this gets injected into all Thrift modules" part somewhere rather than doing it all here. Like maybe, `require('./message').injectInto(this)`?